build.assets/makefiles/master/k8s-master/kube-kubelet.service

```diff
@@ -9,6 +9,7 @@ EnvironmentFile=/run/dns.env
 EnvironmentFile=-/run/kubernetes-reserved.env
 ExecStartPre=/bin/systemctl is-active docker.service
 ExecStart=/usr/bin/kubelet \
+        --root-dir=/var/lib/gravity/planet/kubelet \
```

Comment:
I'd have to change this to `/var/lib/kubelet` (so it's the same as the kubelet's default) when I port the SELinux changes to 6.1.x. The reason I kept the `root-dir` set to the default (and it should be irrelevant for this PR really, as it is just configuration in gravity) is that kubelet hardcodes the default root directory in some paths resulting in issues (well, inconsistency in path handling) as we advertise planet file contexts to SELinux.